evaluating-agents/agent_evals.py

```diff
@@ -57,8 +58,19 @@ async def entrypoint(ctx: JobContext):
 
     await session.start(
         agent=SimpleAgent(),
-        room=ctx.room
+        room=ctx.room,
+        room_input_options=RoomInputOptions(
+            # uncomment to enable Krisp BVC noise cancellation
+            # noise_cancellation=noise_cancellation.BVC(),
+            # listen agents in addition to SIP and standard participants
+            participant_kinds=[
+                rtc.ParticipantKind.PARTICIPANT_KIND_SIP,
+                rtc.ParticipantKind.PARTICIPANT_KIND_STANDARD,
```

Comment:
I think we should remove these, the eval agent should only listen to other agents